source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -44,11 +44,41 @@ matching data is inserted.
 Find
 ----
 
-The ``find()`` method is called on the ``Collection`` object that
-references the collection you want to query. The method accepts a query
-document that describes the documents you want to retrieve. For more
-information on how to specify your query document, see our guide on
-how to :doc:`Specify a Query </fundamentals/crud/query-document>`.
+You can call the ``find()`` method on a ``Collection`` object. The
+method accepts a query document that describes the documents you want to
+retrieve. For more information on how to specify your query document,
+see our guide on how to :ref:`node-fundamentals-query-document`.
+
+.. tip:: No Query Criteria
+   
+   To execute a find operation that has no query criteria, you can
+   pass an empty query or omit the query document in your find
+   method parameters.
+
+   The following operations both return all documents in the
+   ``myColl`` collection:
+
+   .. code-block:: javascript
+
+      await myColl.find(); // no query
+      await myColl.find({}); // empty query
+   
+   If you don't pass a query or pass an empty query
+   to the ``findOne()`` method, the operation returns the first
+   document of a collection.
```

Comment:
[question]

Is this accurate? I thought it returned a random one, not a guaranteed first one.

Reply:
In my testing, it is always the first one, but here's what the Manual says:
Returns one document that satisfies the specified query criteria on the collection or [view](https://www.mongodb.com/docs/manual/core/views/). If multiple documents satisfy the query, this method returns the first document according to the [natural order](https://www.mongodb.com/docs/manual/reference/glossary/#std-term-natural-order) which reflects the order of documents on the disk.

Maybe it's best to be on the safe side and just say a single document